docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -487,10 +487,10 @@ When fields, which are lists (e.g. `scopesSupported`, `responseTypesSupported`),
 
 The client secret can be input directly, or for added security it can be aliased using any of the following features:
```

Comment:
"using any of the following features"

There is only one remaining :-) And I would put MicroProfile into the link, so it's more clear.